### DISCUSSION TITLE
搶先體驗 Standalone Components / Directives / Pipes | 全端開發人員天梯

### DISCUSSION THREAD

**Discussion by giscus** [wellwind/wellwind.github.io#57] [Announcements]
# 搶先體驗 Standalone Components / Directives / Pipes | 全端開發人員天梯

Angular 14 預計會推出一個重磅功能 - 「Standalone Components / Directives / Pipes」！這個功能允許我們「獨立」的使用 components、directives 和 pipes，不用再強制依靠 @NgModule。目前 Angular 14 已經推出 RC.0 了，這個版本內也包含了 Standalone 的開發人員預覽版本（developer preview of standalone），讓我們可以搶先體驗一下這種獨立使用元件的版本，這篇文章就來介紹一下為什麼要使用 Standalone 以及該如何使用。

https://fullstackladder.dev/blog/2022/05/14/angular-14-standalone-components-directives-pipe-preview/

**Comment by eddy1937** [comment]
請問 inline template、 style 是不是沒有 intellisense ?

**Reply by wellwind** [reply]
template 有喔
